io/net/netperf_test.py

```diff
@@ -121,8 +121,12 @@ def test(self):
             if line and 'Throughput' in line.split()[-1]:
                 tput = int(result.stdout.split()[-1].split('.')[0])
                 if tput < self.expected_tp:
-                    self.fail("FAIL: Throughput Actual - %d, Expected - %d"
-                              % (tput, self.expected_tp))
+                    self.fail("FAIL: Throughput Actual - %s, Expected - %s,
+                              Throughput Actual value - %s "
+                              % (str((tput*100)/1000)+'%',
+                                 str((self.expected_tp*100)/1000)+'%',
+                                 str(tput)+'Mb/sec'))
```

Comment:
@smruti77 there is an extra `)` at the end. Remove that.
`str(tput)+'Mb/sec'))` should be `str(tput)+'Mb/sec')`

Reply:
@smruti77 we can print % using
`"%s%%" % string`
Use that